Book-Store-MVC/Controllers/CategoryController.cs

```diff
@@ -8,20 +9,22 @@ namespace Book_Store_MVC.Controllers
     public class CategoryController : Controller
     {
 
-        private readonly BookStoreContext bookStore;
+       
+        private readonly IGenericRepository<Category> genericCategorey;
```

Comment:
use catigroy repo

Reply:
ok

Reply:
regarding  pagination 
i need the  total  to be counted in the GetAll method in order to use it  in the controller

Reply:
Done